### ISSUE TITLE
[Experience] 1.38 Work station has also been bounced off when I hold the tip

### ISSUE THREAD

**Issue by Wandayu** [CryoVR/CryoVR_Vitrobot#23]
**Describe the experience**

What's strange to me is that there's no way to automatically put the tip in the work station now in the new version, it seems to need to be held in the hand at all times, which is probably to be consistent with CP3. But the tip doesn't need to be in the work station all the time, and the progress bar will continue even if the hand and tip are removed after the judgement. Also, because of the collision volume of the hand, the work station would be bounced off all the time, so it would be hard to keep it in one position.

**Comment by smallelephant9516** [comment]
This is also something we meet today, the idea from brenda and I is similar with Dayu, we need the tip to stay in the work station and this will probably solve the problem from Dr. Schmid #16

**Comment by SchubyDong** [comment]
During the last time when we tried to fix a related bug, we enabled the physics simulation of the workstation in several steps to make sure it wouldn't fall in some awkward position. Seemed it causes this problem here. Now we disable the physics simulation, especially for this step. But I will leave this issue as opened status. If any other problems like this (workstation physics simulation)happen in the future. It will be good if you leave comments here.
